controller/cache/info.go

```diff
@@ -37,6 +37,12 @@ func populateNodeInfo(un *unstructured.Unstructured, node *node) {
 			populateIngressInfo(un, node)
 			return
 		}
+	case "networking.k8s.io":
```

Comment:
It would maybe make more sense to extend the previous case with a second value, i.e.
```
case "extensions", "networking.k8s.io":
```
to prevent code redundancy.

Reply:
Makes sense. I'm still a go noob, so such an advice is welcome. :)